2.4/test/html-test-app/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Can we generate this default page and make it show something useful? Like the container environment or help? For example "how I replace this page with my page?"